app/stylesheets/config/type.scss

```diff
@@ -1,3 +1,27 @@
+@import url(https://fonts.googleapis.com/css?family=Open+Sans:400,700,800,300,600);
 @mixin no-margin() {
   margin: 0;
 }
+
+// basic text style in top and bottom tabs
+@mixin tab-text() {
+  font-size: 12px;
+  font-weight: bold;
+  text-align: center;
+}
+
+@mixin transitionAll() {
+  -moz-transition: all 250ms ease;
```

Comment:
Lets leave out browser prefixes from our styles. We can insert them automatically with a processing step on build.